src/components/tooltip/tooltip.ts

```diff
@@ -98,23 +98,20 @@ export default class SlTooltip extends ShoelaceElement {
    */
   @property({ type: Boolean }) hoist = false;
 
+  constructor() {
+    super();
+    // TODO (justinfagnani): does this need to be done in firstUpdated for some
```

Comment:
🤔 perhaps the handlers need the inner shadow dom to be rendered? In which case they should probably just have guard clauses.

cc: @claviska any insight here?